packages/fxa-settings/src/components/Settings/index.tsx

```diff
@@ -22,8 +24,11 @@ export const Settings = ({ account }: { account: AccountData }) => {
 
   const modalHeaderId = 'modal-header-verify-email';
   const modalDescId = 'modal-desc-verify-email';
+  oldPassword('asdzxcasd')
```

Comment:
all the additions in this file are for testing out the hook. it'll all move into whatever change password component we make